tox.ini

```diff
@@ -45,9 +45,7 @@ deps =
     isort
     codespell
 commands =
-    codespell {[vars]all_path}
-    codespell {toxinidir}/*.yaml {toxinidir}/*.ini {toxinidir}/*.md \
-      {toxinidir}/*.toml {toxinidir}/*.txt {toxinidir}/.github
+    codespell . --skip .git --skip .tox --skip build --skip lib --skip venv --skip .mypy_cache
```

Comment:
what is the default dictionary used ? From user locale or US or UK ? Can the dictionary be specified ?